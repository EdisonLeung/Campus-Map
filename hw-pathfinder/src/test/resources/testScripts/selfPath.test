# Tests pathfinding for a node that points to itself

# Create Graph
# ;) heyyy

CreateGraph selfPath

# Add nodes
AddNode selfPath a
AddNode selfPath b

# Add edges
AddEdge selfPath a b 5.0

# test pathfinding
FindPath selfPath a a