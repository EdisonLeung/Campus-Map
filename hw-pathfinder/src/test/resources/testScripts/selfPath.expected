# Tests pathfinding for a node that points to itself

# Create Graph
# ;) heyyy

created graph selfPath

# Add nodes
added node a to selfPath
added node b to selfPath

# Add edges
added edge 5.000 from a to b in selfPath

# test pathfinding
path from a to a:
total cost: 0.000